templates/master/00-master/_base/files/usr-local-bin-etcd-member-add-sh.yaml

```diff
@@ -0,0 +1,72 @@
+filesystem: "root"
+mode: 0755
+path: "/usr/local/bin/etcd-member-add.sh"
+contents:
+  inline: |
+    #!/usr/bin/env bash
+    
+    # example
+    # export SETUP_ETCD_ENVIRONMENT=$(oc adm release info --image-for setup-etcd-environment --registry-config=./config.json)
+    # export KUBE_CLIENT_AGENT=$(oc adm release info --image-for kube-client-agent --registry-config=./config.json)
+    # sudo -E ./etcd-member-add.sh 192.168.1.100 $etcd_name
+    
+    if [[ $EUID -ne 0 ]]; then
+      echo "This script must be run as root"
+      exit 1
+    fi
+    
+    : ${SETUP_ETCD_ENVIRONMENT:?"Need to set SETUP_ETCD_ENVIRONMENT"}
+    : ${KUBE_CLIENT_AGENT:?"Need to set KUBE_CLIENT_AGENT"}
+    
+    usage () {
+        echo 'Active server IP address and etcd name required: ./etcd-member-add.sh 192.168.1.100 $etcd_name'
+        exit 1
+    }
+    
+    if [ "$1" == "" ] || [ "$2" == "" ]; then
+        usage
+    fi
+    
+    RECOVERY_SERVER_IP=$1
```

Comment:
`RECOVERY_SERVER_IP` what is it used for?

Reply:
We need a good server ip  (ip of the one of the active members) for member add.

Reply:
+1